spec.md

```diff
@@ -386,9 +386,10 @@ message GetSupportedVersionsResponse {
 // Specifies a version in Semantic Version 2.0 format.
 // (http://semver.org/spec/v2.0.0.html)
 message Version {
-  uint32 major = 1;  // This field is REQUIRED.
-  uint32 minor = 2;  // This field is REQUIRED.
-  uint32 patch = 3;  // This field is REQUIRED.
+  // The value of these fields must not be negative.
```

Comment:
I'd suggest have one such line per field (in case we add more fields in the future)

Reply:
done